docs/source/index.md

```diff
@@ -1,6 +1,6 @@
 # Welcome to the SimPhoNy docs!
 
-SimPhoNy is an ontology-based open-source Python framework that promotes and enables interoperability between any 3rd-party software tool. Here you can learn more about it.
+SimPhoNy is an ontology-based Python framework that promotes and enables interoperability between any 3rd-party software tool. It is licensed under the open-source [BSD 3-clause](./license.md) license.
```

Comment:
@kysrpex I think I like more the way it was before. Perhaps we have `open source` as a link to `/license.html`.

Reply:
Done. I am not sure if Dirk wanted it to be very explicit or not, but this is easily fixable anyway.